From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Batocera Linux <batocera@linux.com>
Date: Thu, 16 Jan 2026 16:00:00 +0000
Subject: [PATCH] arm64: dts: rockchip: Remove unused hdmirx_ctrler properties for Orange Pi 5 Plus

Remove unused sound-dai-cells, hpd-trigger-level, hdmirx-det-gpios,
and pinctrl properties from disabled hdmirx_ctrler node.

Signed-off-by: Batocera Linux <batocera@linux.com>
---
 arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
index 967bae4..5bc0c45 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
@@ -471,14 +471,6 @@ &hdmi1_in_vp1 {
 /* Should work with at least 128MB cma reserved above. */
 &hdmirx_ctrler {
     status = "disabled";
-
-    #sound-dai-cells = <1>;
-    /* Effective level used to trigger HPD: 0-low, 1-high */
-    hpd-trigger-level = <1>;
-    hdmirx-det-gpios = <&gpio1 RK_PC6 GPIO_ACTIVE_LOW>;
-
-    pinctrl-names = "default";
-    pinctrl-0 = <&hdmim1_rx_cec &hdmim1_rx_hpdin &hdmim1_rx_scl &hdmim1_rx_sda &hdmirx_det>;
 };
 
 &hdptxphy_hdmi0 {
